velox/connectors/clp/ClpConnectorSplit.h

```diff
@@ -24,20 +24,26 @@ struct ClpConnectorSplit : public connector::ConnectorSplit {
   ClpConnectorSplit(
       const std::string& connectorId,
       const std::string& path,
-      std::shared_ptr<std::string> kqlQuery)
+      std::shared_ptr<std::string> kqlQuery,
+      const int type)
       : connector::ConnectorSplit(connectorId),
         path_(path),
-        kqlQuery_(kqlQuery) {}
+        kqlQuery_(std::move(kqlQuery)),
+        type_(static_cast<SplitType>(type)) {}
 
   [[nodiscard]] std::string toString() const override {
     return fmt::format(
-        "CLP Split: path: {}, kqlQuery: {}",
+        "CLP Split: path: {}, kqlQuery: {}, type: {}",
         path_,
-        kqlQuery_ ? *kqlQuery_ : "<null>");
+        kqlQuery_ ? *kqlQuery_ : "<null>",
+        type_ == SplitType::kArchive ? "Archive" : "IR");
   }
 
+  enum class SplitType : int { kArchive, kIR };
```

Comment:
Based on our [naming rules for acronyms](https://www.notion.so/yscope/WIP-Coding-Guidelines-9a308b847a5343958ba3cb97a850be66?source=copy_link#fbcd6418926f41bcaebb9d9d391bea8d), this should be `kIr`, right?